src/radarChart.js

```diff
@@ -40,6 +40,12 @@ function displayRADAR(id, options, $element, layout, data, self) {
 
   //If the supplied maxValue is smaller than the actual one, replace by the max in the data
   var maxValue = Math.max(cfg.maxValue, d3.max(data, function(i){return d3.max(i.map(function(o){return o.value;}));}));
+  var minValue = Math.min(0, d3.min(data, function(i){return d3.min(i.map(function(o){return o.value;}));}));
+
+  if(layout.range === false) {
+    maxValue = isNaN(layout.maxValue) ? 1 : layout.maxValue;
```

Comment:
So, can the user save it as an empty string or string with only spaces? If they can the result is one of the 2 bottom ones in the list above, which evaluates to false and will therefore be used.

The check might have to be a bit more extensive depending on which values layout.maxValue can end up having.

To be a bit more specific, what i'm saying is that if layout.maxValue is null, true, '' or ' ' when this line of code runs, maxValue will be set to that value rather than 1.

Reply:
after testing it manually,, if i entered '' or ' ',, it autoMagically reset it to 1 for maxValue and 0 for minValue and continues depending on that.
i could do something like :
`
maxValue = isNaN(layout.maxValue) || layout.maxValue ==='' || layout.maxValue === ' ' ? 1 : layout.maxValue;
`